nixos/maintainers/scripts/cloudstack/cloudstack-image.nix

```diff
@@ -2,8 +2,6 @@
 
 { config, lib, pkgs, ... }:
 
-with lib;
```

Comment:
Would be nice to have a PR with only those removals, since this one can be merged really fast.

Reply:
I am not following.
My idea is that each commit should be buildable. If I merely remove the `with lib` and merge, obviously it will generate errors like "mkDefault not defined".

Reply:
What do you mean? That this commit is broken them? Because it is clear that a few files only remove `with lib;` and do nothing else.

Reply:
I am talking about only about the files that remove `with lib;` and do nothing else BTW.

Reply:
Good idea.

Reply:
Yes, same idea from me. If we can remove a ``with`` anywhere without having regressions we can immediately merge it. This can also be done treewide to make mistakes in commits unlikely.

This change is uncontroversial for me